sklearn/linear_model/_glm/glm.py

```diff
@@ -610,9 +610,15 @@ def _get_loss(self):
 
 
 class GammaRegressor(_GeneralizedLinearRegressor):
-    """Generalized Linear Model with a Gamma distribution.
+    r"""Generalized Linear Model with a Gamma distribution.
 
     This regressor uses the 'log' link function.
+    Its optimization objective is 
+    
+    .. math::
```

Comment:
For refs, check:
- [GLMs in docs](https://scikit-learn.org/stable/modules/linear_model.html#generalized-linear-regression)
- [Gamma regressor loss](https://github.com/scikit-learn/scikit-learn/blob/af930a9aa4f55361a66051ac9ef151cda3742bf8/sklearn/_loss/loss.py#L671)